support/dynamics/gp/developer-toolkit.md

```diff
@@ -51,40 +49,16 @@ This article also includes information about availability and pricing.
   - XML documentation
   - Optional Windows EAI services
 
-## Availability of the components
+## How to install eConnect
```

Comment:
Hi @theley502, it seems that the installation steps of the other two components are still missing from this article. So, I changed the title to "How to install eConnect".

Reply:
that works